### ISSUE TITLE
Latest release calls apps by their process name

### ISSUE THREAD

**Issue by F3R96** [mydockfinder/mydockfinder#685]
It should say "discord" instead of "update" 
![imagen](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/086f736b-5f81-4222-a9c1-df977fb059de)
Same thing happened with firefox and other apps

**Comment by mydockfinder** [comment]
I didn't understand what it meant




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月8日(星期一) 上午8:20
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)














 
It should say "discord" instead of "update"
 imagen.png (view on web)
 Same thing happened with firefox and other apps

**Comment by F3R96** [comment]
Take a look at the application name at the top, it says update instead of discord. It does the same thing with Firefox, I rename it safari but after a while mdf goes and call it by it's process name, Firefox.

The dock also ignores when I open some apps and they won't appear in it until MDF gets restarted

**Comment by mydockfinder** [comment]
Indeed, I found a bug in the code, which will be fixed in the next version
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月8日(星期一) 上午10:12
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)














 
Take a look at the application name at the top, it says update instead of discord. It does the same thing with Firefox, I rename it safari but after a while mdf goes and call it by it's process name, Firefox.
 
The dock also ignores when I open some apps and they won't appear in it until MDF gets restarted

**Comment by F3R96** [comment]
It's working fine now with the latest beta. I'm using stardock curtains on windows 10 to customize  how some of the explorer windows and some other apps looks, like this: 
![imagen](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/dbff5fe3-5b3a-4c40-8e0c-73207529ea26)
And when I use HDR, some apps are getting a buggy minimize animation. This didn't happen before

**Comment by mydockfinder** [comment]
Update again, last night's update fixed the HDR issue




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月17日(星期三) 凌晨4:11
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
It's working fine now with the latest beta. I'm using stardock curtains on windows 10 to customize some how the explorer windows and some other apps looks like this:
 imagen.png (view on web)
 And when I use HDR, some apps are getting a buggy minimize animation. This didn't happen before
 
&mdash;

**Comment by F3R96** [comment]
I'll give it a look. But maybe this is related to me using stardock curtains app and HDR all together

**Comment by mydockfinder** [comment]
No, it's the reason why the HDR was adjusted in the last beta, which was fixed in last night's update




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月17日(星期三) 上午9:35
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
I'll give it a look. But maybe this is related to me using stardock curtains app and HDR all together
 
&mdash;

**Comment by F3R96** [comment]
I am on the latest beta, and this happens with windows universal apps like microsoft store and calculator
https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/baebcd0d-a699-4d32-95e4-2b619ddf1a6a

**Comment by mydockfinder** [comment]
It's your system theme problem, I guess, it's the window title bar




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月17日(星期三) 上午10:24
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
I am on the latest beta, and this happens with windows universal apps like microsoft store and calculator
 https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/baebcd0d-a699-4d32-95e4-2b619ddf1a6a
 
&mdash;

**Comment by F3R96** [comment]
That's what I was saying. That's the stardock curtains app I been using since a long time. It used to work fine, but hdr makes windows to have that broken animation at minimize stance. Although, when hdr is disabled, the animations work just fine

**Comment by mydockfinder** [comment]
I'll test the&nbsp;stardock curtains



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月17日(星期三) 上午10:32
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
That's what I was saying. That's the stardock curtains app I been using since a long time. It used to work fine, but hdr makes windows to have that broken animation at minimize stance. Although, when hdr is disabled, the animations work just fine
 
&mdash;

**Comment by mydockfinder** [comment]
This app does not support win11...

**Comment by F3R96** [comment]
Nope, it only works on windows 10

**Comment by mydockfinder** [comment]
Does the UWP program display correctly when the window is displayed and the animation is turned off?




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月17日(星期三) 中午1:19
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
Nope, it only works on windows 10
 
&mdash;

**Comment by F3R96** [comment]
Yes, it does display without any problem

**Comment by mydockfinder** [comment]
Preferences - Minimization, whether to check the option to minimize dynamic refresh, you can try to check or uncheck



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月18日(星期四) 凌晨0:21
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
Yes, it does display without any problem
 
&mdash;

**Comment by F3R96** [comment]
Latest release automatically maximizes my firefox window every time my screensaver pops up

**Comment by F3R96** [comment]
Also, launchpad won't allow me to rename my icons

**Comment by F3R96** [comment]
Can you add myfinder menus for Vidaldi browser just like firefox?

**Comment by mydockfinder** [comment]
I can adapt this browser




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月9日(星期四) 凌晨3:12
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
Can you add myfinder menus for Vidaldi browser just like firefox?
 
&mdash;

**Comment by F3R96** [comment]
Thanks! 
One question, is firefox supposed to show the progress bar when downloading something? It use to do it before

**Comment by mydockfinder** [comment]
The progress bar of the icon is the progress of the synchronization system tray icon







------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月16日(星期四) 晚上10:06
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
Thanks!
 One question, is firefox supposed to show the progress bar when downloading something? It use to do it before
 
&mdash;

**Comment by F3R96** [comment]
I see.
Another question, how is it going with the notifications bagde? Like having the number of notifications on dock and that kind of stuff? For example windows taskbar

**Comment by F3R96** [comment]
![imagen](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/9242c9d8-c6b4-4442-9f6e-a64d360cd49c)

**Comment by F3R96** [comment]
I'm using a custom calendar from the workshop that it should look like this
![Captura de pantalla 2024-05-18 114926](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/ec92d62e-d2a2-4672-b2ac-737a0af84305)

but everyday the date changes, the calendar will go back to default font size like this:
![Captura de pantalla 2024-05-18 114813](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/32e987bc-e118-4080-8fee-2d73e1cb4b90)

So in order to keep that custom look I have to manually go to edit calendar icon and click yes.

Here's a video of it

https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/aed6d87d-c448-419e-b217-9c0084e85d6e


And here's a screenshot of the config file:
![imagen](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/c6f48c7a-e54a-49dd-8a30-207040255ff8)

How can I keep the custom calendar without having to manually edit it everyday?

**Comment by mydockfinder** [comment]
Whether you use a theme or not, it is possible that the theme contains the configuration of the calendar, you can open the themes folder, and then find the installed theme, iconsconfig.ini file is not there, if there is a delete config.ini




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月18日(星期六) 晚上10:59
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
I'm using a custom calendar from the workshop that it should look like this
 Captura.de.pantalla.2024-05-18.114926.png (view on web)
 
but everyday the date changes, the calendar will go back to default font size like this:
 Captura.de.pantalla.2024-05-18.114813.png (view on web)
 
So in order to keep that custom look I have to manually go to edit calendar icon and click yes.
 
Here's a video of it
 
https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/aed6d87d-c448-419e-b217-9c0084e85d6e
 
And here's a screenshot of the config file:
 imagen.png (view on web)
 
How can I keep the custom calendar without having to manually edit it everyday?
 
&mdash;

**Comment by F3R96** [comment]
HI! is there any chance you can make the HDR function compatible with stardock curtains? My minimize animation breaks when not using HDR.

Windows 10

**Comment by mydockfinder** [comment]
I don't quite understand if I can shoot a video with phone




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月29日(星期三) 凌晨3:45
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
HI! is there any chance you can make the HDR function compatible with stardock curtains? My minimize animation breaks when not using HDR.
 
Windows 10
 
&mdash;

**Comment by F3R96** [comment]
I'm using stardock curtains for windows 10 which customizes windows to look like mac os (traffic lights).
UWP apps like calculator and Microsoft store are compatible with this app, but using curtains with the HDR option from MDF breaks the minimized animation. What I'm asking is if you can make the HDR feature to be compatible with the stardock curtains app, so it doesn't breaks the animation when both app and feature are enabled.

**Comment by mydockfinder** [comment]
I seem to remember someone reporting this problem, is there only the window title bar when the animation is minimized?




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月29日(星期三) 中午11:51
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
I'm using stardock curtains for windows 10 which customizes windows to look like mac os (traffic lights).
 UWP apps like calculator and Microsoft store are compatible with this app, but using curtains with the HDR option from MDF breaks the minimized animation. What I'm asking is if you can make the HDR feature to be compatible with the stardock curtains app, so it doesn't breaks the animation when both app and feature are enabled.
 
&mdash;

**Comment by F3R96** [comment]
I did on this same thread. 
Yep, when  I minimize an app, all I see is the title bar of the said app without any content visible at the moment of doing it

**Comment by mydockfinder** [comment]
But this problem is not very easy to fix




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月29日(星期三) 中午12:02
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Latest release calls apps by their process name (Issue #685)





 
I did on this same thread.
 Yep, when  I minimize an app, all I see is the title bar of the said app without any content visible at the moment of doing it
 
&mdash;

**Comment by F3R96** [comment]
Hi! Can you test firefox on win 10? after a while this happens, and only fix is to reboot mdf 
![imagen](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/fbfe999c-7359-454c-9024-636c50e632b2)
